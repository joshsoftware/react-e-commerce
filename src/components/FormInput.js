import React from 'react';
import { Input } from 'reactstrap';

const FormInput = ({ type, name, id, placeholder }) => {
  return <Input type={type} name={name} id={id} placeholder={placeholder} />;
};

export default FormInput;
