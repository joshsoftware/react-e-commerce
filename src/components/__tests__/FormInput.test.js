import { render } from '@testing-library/react';
import React from 'react';
import FormInput from '../FormInput';

describe('FormInput Component', () => {
  it('Must return FormInput Component', () => {
    const { asFragment } = render(<FormInput />);
    expect(asFragment()).toMatchSnapshot();
  });
});
