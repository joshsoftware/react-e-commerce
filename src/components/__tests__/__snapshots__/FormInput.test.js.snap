// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput Component Must return FormInput Component 1`] = `
<DocumentFragment>
  <input
    class="form-control"
    type="text"
    value=""
  />
</DocumentFragment>
`;
